framework/src/main/java/org/checkerframework/common/value/util/Range.java

```diff
@@ -152,7 +199,7 @@ public String toString() {
     }
 
     @Override
-    public boolean equals(@Nullable Object obj) {
+    public boolean equals(Object obj) {
```

Comment:
You should not remove the `@Nullable` annotation here. The Nullness Checker will complain about an illegal override. The Nullness Checker isn't run on this code yet, but there is no reason to remove this correct specification.